data/api/apiBlock.go

```diff
@@ -47,6 +60,20 @@ type NotarizedBlock struct {
 	Shard uint32 `json:"shard"`
 }
 
+// EpochStartShardData is a structure that hold information about epoch start meta block shard data
```

Comment:
`// EpochStartShardData is a structure that holds data about the epoch start shard data`

Reply:
done